qa/standalone/mon/misc.sh

```diff
@@ -39,6 +39,8 @@ function TEST_osd_pool_get_set() {
 
     setup $dir || return 1
     run_mon $dir a || return 1
+    run_mgr $dir x || return 1
+    run_osd $dir 0 || return 1
     create_rbd_pool || return 1
```

Comment:
Just wondering why we need to create rbd pool for this test (and probably for some others) at all?